src/components/CardDropdownButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Button } from "components/Button";
+import { EllipsisBtnCopy } from "components/styles/Button";
+import Card from "@leafygreen-ui/card";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+interface Props {
+  disabled?: boolean;
+  setIsVisibleCard?: (v: boolean) => void;
+  loading?: boolean;
+  isVisibleCard?: boolean;
+  cardItems: JSX.Element[];
+  dataCyBtn?: string;
+  dataCyCard?: string;
+}
+
+export const CardDropdownButton = ({
+  disabled = false,
+  loading = false,
+  setIsVisibleCard = function noop(v: boolean) {
+    return;
+  },
+  isVisibleCard = true,
+  cardItems,
+  dataCyBtn = "ellipsis-btn",
+  dataCyCard = "card-dropdown",
+}: Props) => {
+  const toggleCard = () => {
+    setIsVisibleCard(!isVisibleCard);
+  };
+  return (
+    <>
+      <Button
+        size="small"
+        dataCy={dataCyBtn}
+        disabled={disabled}
+        loading={loading}
+        onClick={toggleCard}
+      >
+        <EllipsisBtnCopy>...</EllipsisBtnCopy>
```

Comment:
Leafygreen has an ellipses `<Icon/>` and you can place it in to the button directly 
 by passing in the glyph prop 

https://mongodb.github.io/leafygreen-ui/?path=/story/buttons--default